app/ApplicationBase.php

```diff
@@ -262,8 +262,6 @@ function ($translator, $app) {
             'mongodb://' . $mongdbConfig['default']['host'] . ':' . $mongdbConfig['default']['port'],
             [
                 [
-                    'username' => $mongdbConfig['default']['user'],
-                    'password' => $mongdbConfig['default']['password'],
```

Comment:
So this means that the connection to MongoDB server can be done without authentication?

Reply:
It has always been like this, the credentials are not used. The mongodb instance is only listening on 127.0.0.1, so it is not accessible from outside the machine.
My proposal is to get rid of the credentials so it is at least clear no authentication is used.

Reply:
Thanks for clarifying! Good to go then.

Reply:
Yes, that is the next step. It's only safe to remove them from appconfig if they are not fetched from the code.